src/components/Technologies/CategorySelector/RegistrySelector.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./CategorySelector.module.scss";
+import { Select, ConfigProvider, ThemeConfig, theme } from "antd";
+
+interface RegistrySelectorProps {
+  registries: any[];
+  selected: string[];
+  selectRegistries: (registry: string[]) => void;
+}
+
+export default function RegistrySelector({ registries, selected = [""], selectRegistries }: RegistrySelectorProps) {
+  function getOptions() {
+    return Array.from(registries).map((registry) => {
+      return (
+        <Select.Option key={registry.uid}>
+          {registry.name}
+        </Select.Option>
+      );
+    });
+  }
```

Comment:
why not inline this in the main return?

Reply:
I've seen this pattern repeated everywhere. Its just makes it harder to track what happens in this component